File: 3_private_encaptulaiton.py
class Employee:
    __id = 5212
    def __init__(self,name,salary) -> None:
        self.name = name
        self.salary = salary
    def display_id(self):
        print(Employee.__id)  

    def display(self):
        print(f"{self.name} salary {self.salary}") 
obj = Employee("John's", 50000)
obj.display()
obj.__id = 10
print(obj.__id)

File: 3_private_encaptulaiton.cpp
#include <bits/stdc++.h>
using namespace std;

class Employee {
private:
    int id; 
    string name;
    double salary;

public:
  
    Employee(int empId, string empName, double empSalary) {
        id = empId; 
        name = empName;
        salary = empSalary;
    }

   
    int getId()  {
        return id;
    }


    string getName()  {
        return name;
    }

    double getSalary()  {
        return salary;
    }

    void display()  {
        cout << name << " salary " << salary << endl;
    }
};

int main() {
    Employee obj(5212, "John's", 50000);
    obj.display();
    cout << obj.getId() << endl; 

    return 0;
}
